typings/index.d.ts

```diff
@@ -69,9 +69,9 @@ export interface ViewPagerProps {
      */
     pageMargin?: number;
 
-    style?: ReactNative.StyleProp<ReactNative.ViewStyle>
+    style?: ReactNative.StyleProp<ReactNative.ViewStyle>;
 
-    children?: React.ReactChildren
+    children: React.ReactNode;
```

Comment:
Could you add `?` to have consistency with flow types?  :
https://github.com/react-native-community/react-native-viewpager/blob/master/js/types.js#L102

Reply:
I don't think that is needed as `ReactNode` is union type with case `undefined` itself (see my description of the PR)

Reply:
Even with Flow, shouldn't children be required?

Reply:
I don't think it is unreasonable to have no children? I don't have a particular use case in mind other than for instance populating items dynamically and having transitions/animations for inserting new nodes. As far as I could see in the JS-part of it supports null/undefined cases at least.

Reply:
even if you're populating data from an API, you'd likely be passing an empty array rather than null or undefined, though same argument can be applied to any component. regardless, I'm not talking about passing null or undefined. the question mark marks the prop as optional, which is different from requiring to pass undefined/null explicitly. optional children definitely not correct usage in this case. so the flow types shouldn't have ? either.